### ISSUE TITLE
Crash when exporting a directory via the CLI on 24.4.6

### ISSUE THREAD

**Issue by leuchtum** [jgraph/drawio-desktop#1746]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

You must agree to search and the code of conduct. You must fill in this entire template. If you delete part/all or miss parts out your issue will be closed.

**Describe the bug**
When using the CLI and exporting a directory, drawio crashes. This does not happens, when a single file is exported. Does not happen for versions < 24.4.6

**To Reproduce**
1. Create directory `src` and add a drawio file into it
2. Single file works
```shell
> drawio -x src/dummy.drawio
src/dummy.drawio -> src/dummy.pdf
```
3. Export the directory, it crashes
```shell
> drawio -x src
ReferenceError: stat is not defined
    at file:///Applications/draw.io.app/Contents/Resources/app.asar/src/main/electron.js:603:12
    at Array.forEach (<anonymous>)
    at addDirectoryFiles (file:///Applications/draw.io.app/Contents/Resources/app.asar/src/main/electron.js:600:26)
    at file:///Applications/draw.io.app/Contents/Resources/app.asar/src/main/electron.js:622:6
```

**Expected behavior**
Works on < 24.4.6

**Screenshots**
NA

**draw.io version (In the Help->About menu of the draw.io editor):**
```shell
> drawio --version
24.4.6
```

**Desktop (please complete the following information):**
```shell
> sw_vers
ProductName:		macOS
ProductVersion:		14.4.1
BuildVersion:		23E224
```

**Additional context**
NA

**Comment by apallier** [comment]
Same problem here with 24.4.7 version

**Comment by davidjgraph** [comment]
Fix in 24.4.8, thanks for report.
